tests/UuidTest.php

```diff
@@ -1431,7 +1428,6 @@ public function providePythonTests(): array
                 'clock_seq' => '27b8',
                 'variant' => Uuid::RFC_4122,
                 'version' => Uuid::UUID_TYPE_HASH_MD5,
-                'class' => UuidV3::class,
```

Comment:
Note: I don't think that changing the type of `Uuid::fromString()` is a BC break here, since the interfaced type is still respected